_docs-sources/intro/access-gruntwork/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Access Gruntwork",
+  "position": 2
+}
```

Comment:
We've undertaken a convention of not using the `_category_.json` files. The preference is to define the the Nav and Sidebars in their respective config files.